### DISCUSSION TITLE
Display Media Card + Padding Bottom Config

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1375] [Proposals]
### Goal

Precisamos ter uma opção de aumentar o espaçamento inferior do card para que o conteúdo não fique tão próximo do rodapé.

![image](https://github.com/Telefonica/mistica-design/assets/123083886/6f476322-9798-4f2e-9c69-aa2a0e031f38)

### Context

Com telas grandes o conteúdo ficaria muito embaixo sem, imagem de exemplo acima.

### Impact

Ter a possibilidade de atirar ou desativar um espaçamento maior em Componentes tipo Poster de tela cheia.

Atual
https://tinyl.io/8ztA

Proposta
https://tinyl.io/8ztE

### Discussion Questions

- Podemos ter uma configuração extra de aumentar o espaçamento inferior?
- Caso negativo podemos criar com átomos igual a proposta do playroom?

**Comment by AnaMontes11** [comment]
Olá @BrunoBiagioniNeto,

After thoroughly reviewing your request, given that it's a very specific case in which you just need to increase the space of 100vh, we believe it is best to create it with atoms. 

Furthermore, we also think that conceptually your need is more aligned with the Hero component rather than a card. In that case, we should proceed with phase 2 of the Hero and adjust the padding to increase the bottom spacing, ensuring that the content is not too close to the footer.

Let me know what you think

**Reply by BrunoBiagioniNeto** [reply]
vamos seguir dessa forma então Ana, criando com átomos.
